src/location-distance-utils.c

```diff
@@ -21,31 +21,33 @@
 
 #include "location-distance-utils.h"
 
-static double d2r(double deg)
-{
-	return (deg * M_PI / 180);
-}
+#ifndef M_PI
```

Comment:
Is that really possible?

Reply:
Yes, this is `token guard`.
I got an error on `M_PI` for some reason, plus this code does nothing if `M_PI` is already defined